### ISSUE TITLE
Crashes on windows due to dynamic linking of libgcc and libstdc++

### ISSUE THREAD

**Issue by rorywalsh** [csound/csound#539]
If Csound is loaded via the API through a host software on Windows it can lead to a crashes on both startup and exit. The crashes it seems are being caused by several plugin opcoes that link dynamically to stdc++ and libgcc. One solution is to build statically to these libs by using "-static-libstdc++" "-static-libgcc" and "-static" flags. The problem is that these flags cause build problems on other platforms. Steven suggested adding a condition to make_plugin() but I'm out of my depth here. As it may not be desirable to always build all plugin opcodes with static linking on Windows(file size increases), it would probably be best to add a CMAKE flag for this?

**Comment by gogins** [comment]
My build of Csound for Windows 64 bit CPU architecture using msys2/mingw64, and executed using csound/mingw64/build-mkg.sh, now appears to build all plugin opcodes with static libraries. People should test the latest Csound 6.07 release candidate Windows installer on SourceForge (starting with rc4) and report any problems here.

**Comment by rorywalsh** [comment]
Thanks Mike. It appears I can now build Cabbage using the libraries in this
version without any dll conflicts! I tested CsoundQT too, and it seems to
run fine, although it does not exit properly. Closing it leaves a zombie
process running. If you try to restart it it reports that the application
is already open. Note that I simply tried to run a few examples. CsoundQT
users will need to push it a little to see that it works Ok.
